actors/common/src/main/java/org/sunbird/learner/actors/role/dao/RoleDao.java

```diff
@@ -0,0 +1,14 @@
+package org.sunbird.learner.actors.role.dao;
+
+import java.util.List;
+import org.sunbird.models.role.Role;
+
+public interface RoleDao {
+
+  /**
+   * Get All Records
+   *
+   * @return List of all Roles from Role Table
+   */
+  List<Role> getAllRecords();
+}
```

Comment:
Also, create RoleGroupDao and UrlActionDao with below methods respectively.

RoleGroupDao.getRoleGroups
UrlActionDao.getUrlActions